mlflow/langchain/langchain_tracer.py

```diff
@@ -61,6 +73,21 @@ def __init__(
         self._parent_span = parent_span
         self._run_span_mapping: Dict[str, LiveSpan] = {}
         self._prediction_context = prediction_context
+        self._request_id = None
+
+    def _dump_trace(self) -> str:
```

Comment:
Can we introduce a method which builds a Trace from the dumped str (or does this already exist)? It will be necessary to reconstruct the trace from payload logs of a served model.

Reply:
Let's discuss on the doc and if we really need it I can file a follow-up PR :D